### PR TITLE
chore: Get rid of .localized and replace it with L10n.Localizable struct - P7

### PR DESCRIPTION
**Pull request by agisilaos**
----
#### PR Submission Checklist for internal contributors

- The **PR Title**
  - [ ] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [ ] contains a reference JIRA issue number like `SQPIT-764`
  - [ ] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
  - [ ] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

This PR is following #713, #730, #732, #735, #754, #783, #803, #815

This PR addresses the task of standardizing localization practices across the codebase. We have replaced all instances of the .localized method with the newly generated L10n.Localizable struct, which provides type-safe access to localized strings.

## Changes

Updated localization calls from .localized to `L10n.Localizable` to ensure consistency and safety.
Enhanced the maintainability of localization by utilizing `SwiftGen` to generate `L10n.Localizable`, reducing the risk of typos and simplifying the process of updating localized strings.

**Note**: We're down to 87 results in 30 files

## Affected files

- `Settings+ColorScheme.swift`
- `SettingsPropertyFactory.swift`
- `UIAlertController+DegradedCall.swift`
- `BitRateLabel.swift`
- `UIAlertController+CompanyLogin.swift`
- `ConversationButtonMessageCell.swift`
- `ParticipantsStringFormatter.swift`
- `ConversationTitleView.swift`
- `CameraKeyboardPermissionsCell.swift`
- `ConversationInputBarViewController+Camera.swift`
- `ConversationInputBarViewController+DragAndDrop.swift`
- `ConversationInputBarViewController.swift`
- `ConversationListTopBarViewController.swift`
- `ConversationAction.swift`
- `LocationSelectionViewController.swift`
- `ProfileHeaderViewController.swift`
- `SelfProfileViewController+NewDeviceAlert.swift`
- `SettingsClientViewController.swift`
- `SettingsTableViewController.swift`

----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

**Review by caldrian — APPROVED** [review]

File: wire-ios/Wire-iOS/Sources/UserInterface/Calling/CallDegradationViewController/UIAlertController+DegradedCall.swift [hunk 1]
```diff
@@ -23,26 +23,29 @@ import WireSyncEngine
...
+                L10n.Localizable.Call.Degraded.Ended.Alert.Message.user(user.name ?? "")
+            } else {
+                message = user.isSelfUser ?
+                L10n.Localizable.Call.Degraded.Alert.Message.`self` :
+                L10n.Localizable.Call.Degraded.Alert.Message.user(user.name ?? "")
+            }
+        } else {
+            message = callEnded ?
+            L10n.Localizable.Call.Degraded.Ended.Alert.Message.unknown :
+            L10n.Localizable.Call.Degraded.Alert.Message.unknown
```
Comment (markusfassbender) [inline wire-ios/Wire-iOS/Sources/UserInterface/Calling/CallDegradationViewController/UIAlertController+DegradedCall.swift]: Can this be easily extracted to a `func`?

Comment (markusfassbender) [reply wire-ios/Wire-iOS/Sources/UserInterface/Calling/CallDegradationViewController/UIAlertController+DegradedCall.swift]: Maybe then `title` too, to keep the same level of abstraction in the closure

File: wire-ios/Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/ParticipantsStringFormatter.swift [hunk 2]
```diff
@@ -119,18 +107,29 @@ final class ParticipantsStringFormatter {
...
     }
 
-    init(message: ZMConversationMessage, font: UIFont = .mediumFont, largeFont: UIFont = .largeSemiboldFont, textColor: UIColor = .from(scheme: .textForeground)) {
+    init(
+        message: ZMConversationMessage,
+        font: UIFont = .mediumFont,
+        largeFont: UIFont = .largeSemiboldFont,
+        textColor: UIColor = .from(
+            scheme: .textForeground
+        )
```
Comment (markusfassbender) [inline wire-ios/Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/ParticipantsStringFormatter.swift]: 💅 Having one argument could be a one liner? as you like

**Review by markusfassbender — COMMENTED** [review]
Thank you!

File: wire-ios/Wire-iOS/Sources/UserInterface/Calling/CallDegradationViewController/UIAlertController+DegradedCall.swift (hunk 1 above)
Comment (agisilaos) [reply wire-ios/Wire-iOS/Sources/UserInterface/Calling/CallDegradationViewController/UIAlertController+DegradedCall.swift]: Did the same for title and message, thank you!

File: wire-ios/Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/ParticipantsStringFormatter.swift (hunk 2 above)
Comment (agisilaos) [reply wire-ios/Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/ParticipantsStringFormatter.swift]: Done!

**Review by markusfassbender — APPROVED** [review]

**Pull request merged**